File: SkaiLab.Invoice.Models/UserLicenseInformation.cs
using System;
using System.Collections.Generic;

namespace SkaiLab.Invoice.Models
{
    public class UserLicenseInformation
    {
        public string UserId { get; set; }
        public User User { get; set; }
        public int PlanId { get; set; }
        public DateTime? ExpireDate { get; set; }
        public int SubscriptionId { get; set; }
        public bool IsExpire { get; set;}
        public bool IsTrail { get; set; }
        public bool IsUserHasLicense { get; set; }
        public bool IsAlertRenewLicense { get; set; }
        public bool IsRenew { get; set; }
        public string PaymentDescription { get; set; }
        public bool IsUserCompleteLicense { get; set; }
        
    }
    public class UserLicenseInformationDetail : UserLicenseInformation
    {
        public List<Plan> Plans { get; set; }
        public List<SubscriptionType> SubscriptionTypes { get; set; }
        public string CouponCode { get; set; }
        public double Total { get; set; }
        public double? DiscountRate { get; set; }
        public double TotalAfterDiscount { get; set; }
        public double TaxAmount { get; set; }
        public double TotalIncludeTax { get; set; }
        public long PaymentId { get; set; }
        public double? TotalDiscount { get; set; }
        public bool IsTax { get; set; }
    }
}

File: SkaiLab.Invoice/ClientApp/src/app/models/user-license-information.ts

import { User } from "./user";

export class UserLicenseInformation{
    userId:string;
    user:User;
    planId:number;
    expireDate:Date;
    subscriptionId:number;
    isExpire:boolean;
    isTrail:boolean;
    isUserHasLicense:boolean;
    isUserCompleteLicense:boolean;
    isAlertRenewLicense:boolean;
    constructor(){
       this.userId="";
       this.user=new User();
        this.planId=0;
        this.expireDate=new Date();
        this.subscriptionId=0;
        this.isExpire=false;
        this.isTrail=false;
        this.isUserHasLicense=false;
        this.isUserCompleteLicense=false;
        this.isAlertRenewLicense=false;
    }
}